application/src/test/java/org/opentripplanner/model/TripTimeOnDateTest.java

```diff
@@ -17,7 +17,7 @@ void gtfsSequence() {
     var testModel = TimetableRepositoryForTest.of();
     var pattern = testModel.pattern(TransitMode.BUS).build();
     var trip = TimetableRepositoryForTest.trip("123").build();
-    var stopTimes = testModel.stopTimesEvery5Minutes(3, trip, T11_00);
+    var stopTimes = testModel.stopTimesEvery5Minutes(3, trip, "11:00");
```

Comment:
Out of curiosity why make this change?

Reply:
Thomas has requested it here: https://github.com/opentripplanner/OpenTripPlanner/pull/6250#discussion_r1853854631